examples/bin/cluster-syntheticcontrol.sh

```diff
@@ -75,11 +75,21 @@ if [ "$HADOOP_HOME" != "" ] && [ "$MAHOUT_LOCAL" == "" ]; then
     echo "DFS is healthy... "
     echo "Uploading Synthetic control data to HDFS"
     $DFSRM ${WORK_DIR}/testdata
-    $DFS -mkdir ${WORK_DIR}/testdata
+    $DFS -mkdir -p ${WORK_DIR}/testdata
```

Comment:
Wasn't aware that was not compatible with Hadoop 1.  However, it's already used in some other examples so those presumably won't work with Hadoop 1 either?

```
>grep DFS *.sh | grep mkdir
classify-20newsgroups.sh:    $DFS -mkdir -p ${WORK_DIR}
classify-20newsgroups.sh:    $DFS -mkdir ${WORK_DIR}/20news-all
classify-wikipedia.sh:    $DFS -mkdir -p ${WORK_DIR}
cluster-reuters.sh:  $DFS -mkdir -p $WORK_DIR
cluster-reuters.sh:        $DFS -mkdir -p ${WORK_DIR}/
cluster-reuters.sh:        $DFS -mkdir ${WORK_DIR}/reuters-sgm
cluster-reuters.sh:        $DFS -mkdir ${WORK_DIR}/reuters-out
cluster-syntheticcontrol.sh:    $DFS -mkdir -p ${WORK_DIR}/testdata
```